documentation/book/con-resource-kafka.adoc

```diff
@@ -0,0 +1,810 @@
+      type: persistent-claim
+      size: 1Gi
+      class: my-storage-class
+    # ...
+----
+
+Lastly, a `selector` can be used in order to select a specific labelled persistent volume which provides some needed features like an SSD.
+
+.Example fragment of a `Kafka` resource configuring Kafka with _match labels_ selector
+[source,json]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    # ...
+    storage:
+      type: persistent-claim
+      size: 1Gi
+      selector:
+        matchLabels:
+          "hdd-type": "ssd"
+      deleteClaim: true
+    # ...
+----
+
+When the _persistent-claim_ is used, other than the resources already described in the xref:kafka-config-map-details-{context}[Kafka] section, the following resources are generated :
+
+`data-[cluster-name]-kafka-[idx]`::
+Persistent Volume Claim for the volume used for storing data for the Kafka broker pod `[idx]`.
+
+`data-[cluster-name]-zookeeper-[idx]`::
+Persistent Volume Claim for the volume used for storing data for the Zookeeper node pod `[idx]`.
+
+See xref:type-EphemeralStorage[`EphemeralStorage` type v1alpha1 kafka.strimzi.io] and xref:type-PersistentClaimStorage[`PersistentClaimStorage` type v1alpha1 kafka.strimzi.io] for more information.
+
+[id='metrics-{context}]
+== Metrics
+
+{ProductName} uses the link:https://github.com/prometheus/jmx_exporter[Prometheus JMX exporter^] in order to expose metrics on each node.
+It is possible to configure a `metrics` object in the `kafka` and `zookeeper` objects in `Kafka` resources, and likewise a `metrics` object in the `spec` of `KafkaConnect` resources.
+In all cases the `metrics` object should be the configuration for the JMX exporter.
+You can find more information on how to use it in the corresponding GitHub repo.
+
+For more information about using the metrics with Prometheus and Grafana, see xref:metrics[Metrics]
+
+[id='logging-examples-{context}']
+== Logging
+The `logging` field allows the configuration of loggers. These loggers for Zookeeper and Kafka are available in the xref:spec.zookeeper.logging-{context}[`spec.zookeeper.logging`] and xref:spec.kafka.logging-{context}[`spec.kafka.logging`] sections respectively.
+
+The setting can be done in two different ways. Either by specifying the loggers and their levels directly or by using a custom configuration map. The following is an example of loggers level setting:
+
+[source,yaml]
+----
+  logging:
+    type: inline
+    loggers:
+      logger.name: "INFO"
+----
+The `INFO` can be replaced with any log4j logger level. The available logger levels are `INFO`, `ERROR`, `WARN`, `TRACE`, `DEBUG`, `FATAL` or `OFF`.
+For information about log levels, see link:https://logging.apache.org/log4j/2.x/manual/customloglevels.html[log4j manual^].
+
+[source,yaml]
+----
+  logging:
+    type: external
+    name: customConfigMap
+----
+
+When using external ConfigMap remember to place your custom ConfigMap under `log4j.properties` key.
```

Comment:
_remember to place your custom ConfigMap under_ -> _remember to place your custom configuration under_